actions/insert.py

```diff
@@ -20,7 +21,7 @@ def insert(self, table, data):
         columns = self._list_to_string(data.keys(), quotes=False)
         values = self._list_to_string(data.values())
 
-        query = "INSERT INTO {} ({}) VALUES ({})".format(table.encode('utf-8'), columns, values)
+        query = "INSERT INTO {} ({}) VALUES ({})".format(six.ensure_str(table.encode('utf-8')), columns, values)
```

Comment:
This line is failing the build because of `actions/insert.py:24:101: E501 line too long (112 > 100 characters)`

probably easy enough to just turn `six.ensure_str(table.encode('utf-8'))` into its own line